lib/evss/gi_bill_status/service.rb

```diff
@@ -7,6 +7,29 @@ module GiBillStatus
     class Service < EVSS::Service
       configuration EVSS::GiBillStatus::Configuration
 
+      OPERATING_ZONE = 'Eastern Time (US & Canada)'
+      OPERATING_HOURS = {
+        start: 6,
+        end: 22,
+        saturday_end: 19
+      }.freeze
+
+      def self.within_scheduled_uptime?
+        current_time = Time.now.in_time_zone(OPERATING_ZONE)
+        if current_time.saturday?
```

Comment:
return will be the result of the if statement if it's at the end of a method, and could use a range to check it the time is covered:
``` ruby
if current_time.saturday?
  (OPERATING_HOURS[:start]..OPERATING_HOURS[:saturday_end]).cover?(current_time.hour)
else
  (OPERATING_HOURS[:start]..OPERATING_HOURS[:end]).cover?(current_time.hour)
end
```
or in ternary form
```ruby
current_time.saturday? ?
  (OPERATING_HOURS[:start]..OPERATING_HOURS[:saturday_end]).cover?(current_time.hour) :
  (OPERATING_HOURS[:start]..OPERATING_HOURS[:end]).cover?(current_time.hour)
```